Source/Fuse.Reactive.Expressions/MathFunctions.uno

```diff
@@ -33,4 +34,462 @@ namespace Fuse.Reactive
 			return "max(" + Left + ", " + Right + ")";
 		}
 	}
+	
+	[UXFunction("mod")]
+	public sealed class Mod : BinaryOperator
+	{
+		[UXConstructor]
+		public Mod([UXParameter("Left")] Expression left, [UXParameter("Right")] Expression right): base(left, right) {}
+		protected override object Compute(object left, object right)
+		{
+			return Math.Mod( Marshal.ToFloat(left), Marshal.ToFloat(right) );
+		}
+
+		public override string ToString()
+		{
+			return "mod(" + Left + ", " + Right + ")";
+		}
+	}
+
+	[UXFunction("even")]
+	/** True if the rounded value is even, false otherwise*/
+	public sealed class Even : UnaryOperator
+	{
+		[UXConstructor]
+		public Even([UXParameter("Operand")] Expression operand): base(operand) {}
+		protected override object Compute(object operand)
+		{
+			float v = 0;
+			if (!Marshal.TryToType<float>(operand, out v))
+				return null;
+				
+			var q = (int)Math.Round(v);
+			return q % 2 == 0;
+		}
+
+		public override string ToString()
+		{
+			return "even(" + Operand +  ")";
+		}
+	}
+	
+	[UXFunction("odd")]
+	/** True if the rounded value is odd, false otherwise*/
+	public sealed class Odd : UnaryOperator
+	{
+		[UXConstructor]
+		public Odd([UXParameter("Operand")] Expression operand): base(operand) {}
+		protected override object Compute(object operand)
+		{
+			float v = 0;
+			if (!Marshal.TryToType<float>(operand, out v))
+				return null;
+				
+			var q = (int)Math.Round(v);
+			return q % 2 != 0;
+		}
+
+		public override string ToString()
+		{
+			return "odd(" + Operand +  ")";
+		}
+	}
+
+	[UXFunction("alternate")]
+	/**
+		Alternate between true/false values for ranges of integers.
+		
+			alternate( value, groupSize )
+			
+		Input values are rounded to the nearest integer.
+		
+		Example:
+		
+			alternate( value, 3 )
+			
+		This will yield true for values 0,1,2, false for 3,4,5, true for 6,7,8, false for 9,10,11, etc.
+	*/
+	public sealed class Alternate : BinaryOperator
+	{
+		[UXConstructor]
+		public Alternate([UXParameter("Left")] Expression left, [UXParameter("Right")] Expression right): base(left, right) {}
+		protected override object Compute(object left, object right)
+		{
+			float fvalue = 0;
+			float fgroup = 0;
+			if (!Marshal.TryToType<float>(left, out fvalue) ||
+				!Marshal.TryToType<float>(right, out fgroup))
```

Comment:
The indentation is normal here. It's one additional tab from the previous line, since it's still part of that line. The other option would be to start with `if(` and have all conditions on subsequent lines for alignment. I'm not sure that's better though.